### DISCUSSION TITLE
Build iPhone simulator error on mac chip M1

### DISCUSSION THREAD

**Discussion by antn9x** [axmolengine/axmol#1398] [Q&A] [answered]
My build command: `cmake -S . -B ios-build \
  -GXcode -DCMAKE_TOOLCHAIN_FILE=$AX_ROOT/1k/ios.cmake \
  -DPLAT=iOS -DARCHS=arm64 `
then Run on Xcode and get this error message
`In ~/Documents/github/axmol/thirdparty/zlib/prebuilt/ios/libz.a(adler32.o), building for iOS Simulator, but linking in object file built for iOS, file '~/Documents/github/axmol/thirdparty/zlib/prebuilt/ios/libz.a' for architecture arm64`
Anyone can help me to fix this?

**Comment by halx99** [comment]
from DevSetup.md:

ios:
- for device: axmol build -p ios -a arm64 -c, generate xcodeproj, open by xcode to setup code sign cert and build
- for simulator: axmol build -p ios -a x64

**Reply by antn9x** [reply]
I used `axmol build -p ios -a x64` 
<img width="365" alt="Screenshot 2023-10-21 at 23 19 03" src="https://github.com/axmolengine/axmol/assets/2052604/b68566a5-3df6-4371-a695-ff7ce7f587fd">
There is no simulator device when run.

**Reply by rh101** [reply, marked as answer]
The x64 simulators are not being displayed in your case.  Native arm64 simulators are now available, and they are the ones displayed by default, but since your build is for x64, you won't see the compatible simulators in the list.  To show the x64 simulators, navigate the Xcode menu and change the relevant option:

`Product -> Destination -> Destination Architectures -> Select "Show Both"`

**Reply by antn9x** [reply]
Thank you, it work. Please mark as solution.
